pkg/webhook/database.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhook
+
+import "context"
+
+// DatastoreClient adheres to the interaction with a BigQuery instance.
+type DatastoreClient interface {
```

Comment:
nit: some suggestion for conciseness

You could just name it `Datastore`. You could also move this to `server.go` where the interface is used.